debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -564,4 +571,49 @@ public String quotedTableIdString(TableId tableId) {
     private String replaceDatabaseNamePlaceholder(String sql, String databaseName) {
         return sql.replace(DATABASE_NAME_PLACEHOLDER, databaseName);
     }
+
+    public void callbackOnReadingNewChangeTable(SqlServerPartition partition, ChangeTable table) throws SQLException {
+        final String query = replaceDatabaseNamePlaceholder(START_READING_CHANGE_TABLE,
+                partition.getDatabaseName());
+        final String startReadingChangeTableStmt = String.format(query, table.getCaptureInstance());
```

Comment:
The connector should use parametrized statements where necessary instead of concatenating strings.